From c2e41a17ce2ffa18451c020d5d85ee929d0a0323 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Sat, 27 Jun 2020 12:23:12 +0800
Subject: [PATCH 4/8] crypto: caam/qi - drop unused label out_put_dev
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

To fix below warning:
drivers/crypto/caam/caamalg_qi.c:3172:1: warning: label 'out_put_dev' defined but not used [-Wunused-label]

3172 | out_put_dev:

      | ^~~~~~~~~~~

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/crypto/caam/caamalg_qi.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/crypto/caam/caamalg_qi.c b/drivers/crypto/caam/caamalg_qi.c
index 6f8040e90193..f10715cea4f4 100644
--- a/drivers/crypto/caam/caamalg_qi.c
+++ b/drivers/crypto/caam/caamalg_qi.c
@@ -3169,7 +3169,6 @@ int caam_qi_algapi_init(struct device *ctrldev)
 	if (registered)
 		dev_info(ctrldev, "algorithms registered in /proc/crypto\n");
 
-out_put_dev:
 	put_device(ctrldev);
 	return err;
 }
-- 
2.17.1

